src/webview/src/main.css

```diff
@@ -1,6 +1,35 @@
sbD0iI0EwQjJGRiIvPgo8L2c+CjwvZz4KPC9nPgo8L2c+CjwvZz4KPGRlZnM+CjxsaW5lYXJHcmFkaWVudCBpZD0icGFpbnQwX2xpbmVhcl81MjM3XzEyMDYyNSIgeDE9IjI1Ny4wOTIiIHkxPSI1NTMuNjYzIiB4Mj0iMjU3LjA5MiIgeTI9IjI3NyIgZ3JhZGllbnRVbml0cz0idXNlclNwYWNlT25Vc2UiPgo8c3RvcCBzdG9wLWNvbG9yPSIjRjVGN0ZGIi8+CjxzdG9wIG9mZnNldD0iMSIgc3RvcC1jb2xvcj0iI0Y1RjdGRiIvPgo8L2xpbmVhckdyYWRpZW50Pgo8bGluZWFyR3JhZGllbnQgaWQ9InBhaW50MV9saW5lYXJfNTIzN18xMjA2MjUiIHgxPSI3NDIuOTgyIiB5MT0iNTUzLjY2MyIgeDI9Ijc0Mi45ODIiIHkyPSIyNzciIGdyYWRpZW50VW5pdHM9InVzZXJTcGFjZU9uVXNlIj4KPHN0b3Agc3RvcC1jb2xvcj0iI0Y1RjdGRiIvPgo8c3RvcCBvZmZzZXQ9IjEiIHN0b3AtY29sb3I9IiNGNUY3RkYiLz4KPC9saW5lYXJHcmFkaWVudD4KPGxpbmVhckdyYWRpZW50IGlkPSJwYWludDJfbGluZWFyXzUyMzdfMTIwNjI1IiB4MT0iMjU3LjA5MiIgeTE9IjAuMzM3MjE5IiB4Mj0iMjU3LjA5MiIgeTI9IjI3NyIgZ3JhZGllbnRVbml0cz0idXNlclNwYWNlT25Vc2UiPgo8c3RvcCBzdG9wLWNvbG9yPSIjRjVGN0ZGIi8+CjxzdG9wIG9mZnNldD0iMSIgc3RvcC1jb2xvcj0iI0Y1RjdGRiIvPgo8L2xpbmVhckdyYWRpZW50Pgo8bGluZWFyR3JhZGllbnQgaWQ9InBhaW50M19saW5lYXJfNTIzN18xMjA2MjUiIHgxPSI3NDIuOTgyIiB5MT0iMC4zMzcyMTkiIHgyPSI3NDIuOTgyIiB5Mj0iMjc3IiBncmFkaWVudFVuaXRzPSJ1c2VyU3BhY2VPblVzZSI+CjxzdG9wIHN0b3AtY29sb3I9IiNGNUY3RkYiLz4KPHN0b3Agb2Zmc2V0PSIxIiBzdG9wLWNvbG9yPSIjRjVGN0ZGIi8+CjwvbGluZWFyR3JhZGllbnQ+CjxsaW5lYXJHcmFkaWVudCBpZD0icGFpbnQ0X2xpbmVhcl81MjM3XzEyMDYyNSIgeDE9IjI1Ny4wOTIiIHkxPSIxMTA3LjY2IiB4Mj0iMjU3LjA5MiIgeTI9IjgzMSIgZ3JhZGllbnRVbml0cz0idXNlclNwYWNlT25Vc2UiPgo8c3RvcCBzdG9wLWNvbG9yPSIjRjVGN0ZGIi8+CjxzdG9wIG9mZnNldD0iMSIgc3RvcC1jb2xvcj0iI0Y1RjdGRiIvPgo8L2xpbmVhckdyYWRpZW50Pgo8bGluZWFyR3JhZGllbnQgaWQ9InBhaW50NV9saW5lYXJfNTIzN18xMjA2MjUiIHgxPSI3NDIuOTgyIiB5MT0iMTEwNy42NiIgeDI9Ijc0Mi45ODIiIHkyPSI4MzEiIGdyYWRpZW50VW5pdHM9InVzZXJTcGFjZU9uVXNlIj4KPHN0b3Agc3RvcC1jb2xvcj0iI0Y1RjdGRiIvPgo8c3RvcCBvZmZzZXQ9IjEiIHN0b3AtY29sb3I9IiNGNUY3RkYiLz4KPC9saW5lYXJHcmFkaWVudD4KPGxpbmVhckdyYWRpZW50IGlkPSJwYWludDZfbGluZWFyXzUyMzdfMTIwNjI1IiB4MT0iMjU3LjA5MiIgeTE9IjU1NC4zMzciIHgyPSIyNTcuMDkyIiB5Mj0iODMxIiBncmFkaWVudFVuaXRzPSJ1c2VyU3BhY2VPblVzZSI+CjxzdG9wIHN0b3AtY29sb3I9IiNGNUY3RkYiLz4KPHN0b3Agb2Zmc2V0PSIxIiBzdG9wLWNvbG9yPSIjRjVGN0ZGIi8+CjwvbGluZWFyR3JhZGllbnQ+CjxsaW5lYXJHcmFkaWVudCBpZD0icGFpbnQ3X2xpbmVhcl81MjM3XzEyMDYyNSIgeDE9Ijc0Mi45ODIiIHkxPSI1NTQuMzM3IiB4Mj0iNzQyLjk4MiIgeTI9IjgzMSIgZ3JhZGllbnRVbml0cz0idXNlclNwYWNlT25Vc2UiPgo8c3RvcCBzdG9wLWNvbG9yPSIjRjVGN0ZGIi8+CjxzdG9wIG9mZnNldD0iMSIgc3RvcC1jb2xvcj0iI0Y1RjdGRiIvPgo8L2xpbmVhckdyYWRpZW50Pgo8Y2xpcFBhdGggaWQ9ImNsaXAwXzUyMzdfMTIwNjI1Ij4KPHJlY3Qgd2lkdGg9IjEwMDAiIGhlaWdodD0iMjc4IiBmaWxsPSJ3aGl0ZSIgdHJhbnNmb3JtPSJtYXRyaXgoMSAwIDAgLTEgMCA1NTQpIi8+CjwvY2xpcFBhdGg+CjxjbGlwUGF0aCBpZD0iY2xpcDFfNTIzN18xMjA2MjUiPgo8cmVjdCB3aWR0aD0iMTAwMCIgaGVpZ2h0PSIyNzgiIGZpbGw9IndoaXRlIi8+CjwvY2xpcFBhdGg+CjxjbGlwUGF0aCBpZD0iY2xpcDJfNTIzN18xMjA2MjUiPgo8cmVjdCB3aWR0aD0iMTAwMCIgaGVpZ2h0PSIyNzgiIGZpbGw9IndoaXRlIiB0cmFuc2Zvcm09Im1hdHJpeCgxIDAgMCAtMSAwIDExMDgpIi8+CjwvY2xpcFBhdGg+CjxjbGlwUGF0aCBpZD0iY2xpcDNfNTIzN18xMjA2MjUiPgo8cmVjdCB3aWR0aD0iMTAwMCIgaGVpZ2h0PSIyNzgiIGZpbGw9IndoaXRlIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgwIDU1NCkiLz4KPC9jbGlwUGF0aD4KPC9kZWZzPgo8L3N2Zz4K");
```

Comment:
Hmm can't we do it with another way to avoid this huge base64 inline code?

Reply:
I think It has to be that way, since it is built as one file.
Maybe it is possible to export it, but it'll be just a bit uglier. The whole CSS may be refactored.